cfdm/decorators.py

```diff
@@ -22,7 +22,7 @@
 def _inplace_enabled(operation_method=None, *, default=False):
     '''A decorator enabling operations to be applied in-place.
 
-    If the decorated method has keyword argument *inplace* being equal
+    If the decorated method has keyword argument ``inplace`` being equal
```

Comment:
I think we need a policy on **\*** vs. **\`\`** vs. **\`**!

The unwritten policy that I've been following is to use **\*** for parameters in the abstract (rendering as italics), and **\`\`** for anything that might by typed or that appears on the screen, and  **\`** being reserved for functions, classes, methods, and attributes (rendering as (usually) clickable links in the online docs).

By this reckoning, inplace would be `*inplace*`

This is a framework that has evolved organically over many years, largely driven by the tech in use at any one time to render the docs, so very happy to review this never-reviewed procedure!

Perhaps it's best to use `*inplace*` for now? Then if we were to decide on a new approach, it would be easier to find and change.

Reply:
Ah, sorry I did not realise/recall this was a somewhat controversial change, I made it in the spur of the moment because it the italicised keyword didn't seem right at an eyeball! But now you point it out I have a vague recollection of you mentioning this early in the year, so I think there was a policy and just I overlooked it :grimacing:

I'm happy to continue with the "unwritten policy", I'll pop in a commit now to fix this case I defaced and also the equivalent case of the new display argument.

(Also, sorry if you wanted to do a review. I didn't assign you or ask if you wanted to review this because this PR was pretty straightforward.)

Reply:
Sorted here: 548584e38ea189cd5cbc81dda0179c0fed2f3a7e. I have made a mental note to continue with the asterisk/backtick policy you outlined above (assuming you aren't desperate to change it).

Reply:
Hi Sadie, that's all good. I'm not at desparate to change it - if/when the current style isn't doing what we want any more is probably the time to review it.